ports/stm32/boards/NUCLEO_G474RE/mpconfigboard.mk

```diff
@@ -0,0 +1,21 @@
+DEBUG = 0
+
+# MCU settings
+MCU_SERIES = g4
+CMSIS_MCU = STM32G474xx
+MICROPY_FLOAT_IMPL = double
```

Comment:
Does this MCU have hardware double FP?

Reply:
Good point. It has single precision FP. Changed it.